web/src/components/Avatar/Avatar.tsx

```diff
@@ -2,6 +2,7 @@ import * as AvatarPrimitive from '@radix-ui/react-avatar';
 import * as React from 'react';
 
 import { cn } from '@/lib/utils';
+import { Skeleton } from '..';
```

Comment:
How about '@/components' here? I am exporting all the components from 'components/index.ts'

Reply:
For sure! That's my VS Code's autoimport. Will fix